gcloud-java-dns/src/main/java/com/google/gcloud/testing/OptionParsersAndExtractors.java

```diff
@@ -0,0 +1,292 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.testing;
+
+import com.google.api.services.dns.model.Change;
+import com.google.api.services.dns.model.ManagedZone;
+import com.google.api.services.dns.model.Project;
+import com.google.api.services.dns.model.ResourceRecordSet;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Utility helpers for LocalDnsHelper.
+ */
+class OptionParsersAndExtractors {
```

Comment:
I think this class could be renamed to `OptionParsers` (codacy doesn't like long variable names).  In general, while a few of the codacy complaints can be ignored (e.g. it complains that LocalDnsHelper is too long, but that's consistent with our other class names), a lot of those errors look legitimate and simple to fix.